packages/example-api-gateway/package.json

```diff
@@ -37,6 +38,7 @@
     "@aloxide/demux": "^0.1.0",
     "@aloxide/logger": "^0.1.0",
     "@aloxide/model": "^0.1.0",
+    "debug": "^4.1.1",
```

Comment:
@tiendq please try to reduce dependency as less as possible. I don't think we should have the `debug` package as a must.